securitygroups.tf

```diff
@@ -28,21 +26,16 @@ resource "aws_security_group" "web" {
   vpc_id      = aws_vpc.vpc.id
 
   ingress {
-    from_port = 22
-    to_port   = 22
-    protocol  = "tcp"
-    cidr_blocks = [
-      "0.0.0.0/0"
-    ]
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["<cidr>"]
```

Comment:
**Tenable.cs remediation**
 
**[LOW]: Ensure SSH (TCP,22) is not accessible by a public CIDR block range**
 
**Vulnerability**
'SSH' (TCP,22) is accessible by a CIDR block range. Exposed ports may allow attackers to access the application listening on that port.
 
**Remediation**
In AWS Console - 
 1. Sign in the AWS Console and go to the Amazon VPC console.
 2. In the navigation pane, select Security Groups.
 3. For each security group, perform the following actions:
      a) Select the security group.
      b) Select the Inbound Rules.
      c) Identify the rules to be removed.
      d) Remove the required rules.
 4. Click Save.

source: tenablecs-